src/tallies/tally_scoring.cpp

```diff
@@ -1902,8 +1902,8 @@ score_general_mg(const Particle* p, int i_tally, int start_index,
           for (auto i = 0; i < p->n_bank_; ++i) {
             auto i_bank = simulation::fission_bank.size() - p->n_bank_ + i;
             const auto& bank = simulation::fission_bank[i_bank];
-            auto g = bank.delayed_group;
-            if (g != 0) {
+            auto g = bank.delayed_group - 1;
+            if (g != -1) {
```

Comment:
Note to self: make this g a dg or something to denote delayed group vice energy group. @paulromano, since I'll put together a PR shortly on this, no need to attack this here unless you have to revise code for any other comments I suppose.

Reply:
I'll leave it to you so you can make all relevant instances consistent.